src/protocol/webdriver-bidi.ts

```diff
@@ -0,0 +1,1808 @@
+/**
```

Comment:
add a comment this file is generated and instructions how to re-generate?

Reply:
I will do this in another PR (I don't thinking mixing migration with generation will be easy to review).